### DISCUSSION TITLE
Lesson 7 -  test with console.log reverted

### DISCUSSION THREAD

**Discussion by vasilsariev** [Cyfrin/foundry-full-course-cu#989] [Q&A] [answered]
Hey guys, I'm doing the command : 
```
forge test --mt testPriceFeedVersionIsAccurate -vvv --fork-url $SEPOLIA_RPC_URL
```

It works fine the way Patrick does it, but i tried logging the version with this line: 
```solidity
console.log(version); 
```

It looks like this : 

```solidity
function testPriceFeedVersionIsAccurate() public {
        uint256 version = fundMe.getVersion();
        console.log(version);
        assertEq(version, 4);
    }
```

It fails with the revert error. Why would that be?

**Comment by shikhar229169** [comment, marked as answer]
Can you provide the error with verbosity -vvvv
Also, ensure that the price feeds address and sepolia rpc url is correct.

**Comment by vasilsariev** [comment]
Hey @shikhar229169,  thank you for the advice. It turns out that probably the rpc url changed - I replaced it with the new one from the Alchemy app and now it works and the console.log command prints out the version.
